.github/workflows/haskell-ci.yml

```diff
@@ -11,113 +11,108 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macOS-latest, windows-latest]
-        cabal: ["3.2"]
+        os:
+          - ubuntu-latest
+          - macOS-latest
+          - windows-latest
+        cabal: [3.6]
         ghc:
-          - "8.4.4"
```

Comment:
maybe you can already put that we're dropping this into change log so we don't forget https://github.com/Holmusk/elm-street/blob/master/CHANGELOG.md

Reply:
Good point, updated in the next commit. Also mentioned other not-yet-released changes.